content/pages/getting-started-with-github-pages/troubleshooting-404-errors-for-github-pages-sites.md

```diff
@@ -69,5 +69,6 @@ Check whether your repository meets the following requirements.
 * The branch you are using to publish your site must be the `main` or default branch.
 * The repository must have a commit pushed to it by someone with admin permissions for the repository, such as the repository owner.
 * Switching the repository's visibility from public to private or vice versa will change the URL of your {% data variables.product.prodname_pages %} site, which will result in broken links until the site is rebuilt.
+* Check if your GitHub Pro, GitHub Team, GitHub Enterprise Cloud, or GitHub Enterprise Server access has expired if you are using a private repository for the Github Page Site. If so, please make sure to renew it or change your repository's visibility to public.
```

Comment:
```suggestion
* Check if your {% data variables.product.prodname_pro %}, {% data variables.product.prodname_team %}, {% data variables.product.prodname_ghe_cloud %}, or {% data variables.product.prodname_ghe_server %} subscription is still active if you are using a private repository for the {% data variables.product.prodname_pages %} site. Otherwise please make sure to renew it or change your repository's visibility to public.
```

Reply:
Enterprise Server doesn't apply in this situation due to how it's installed, so I've removed that. I also have a few suggestions on the wording/phrasing in case that's helpful.

```suggestion 
* If you are using a private repository for the {% data variables.product.prodname_pages %} site, please check if your {% data variables.product.prodname_pro %}, {% data variables.product.prodname_team %}, or {% data variables.product.prodname_ghe_cloud %} subscription is still active. If you renew the plan, the {% data variables.product.prodname_pages %} site will be automatically re-deployed. Otherwise, you can change your repository's visibility to public to continue using {% data variables.product.prodname_pages %} for free.
```

Reply:
@Yuchen-Song Hello! 👋 Once you have committed @tcbyrd's suggestion above, we will be happy to ge this merged! 💛

Reply:
Hi @nguyenalex836, I have committed the suggested wording and formatting above, thank you all for your comments :)